tests/test_instance_config.py

```diff
@@ -4,6 +4,13 @@
 
 import flask
 
+from config_module_app import app as config_module_app
+from config_package_app import app as config_package_app
+from namespace.package2 import app as namespace_package2_app
+from site_app import app as site_app
+from installed_package import app as installed_package_app
+import site_package as site_package
```

Comment:
Love this changes. It follows standard community practices and improves readable. Good job!